### ISSUE TITLE
Fastlane Z3 issues

### ISSUE THREAD

**Issue by 10MARC** [mbtaylor1982/ReSDMAC#16]
Just an observation. I have a Fastlane Z3 with 64 MB of RAM and fast SCSI. With my V2 DMAC the Fast SCSI on the card would corrupt my install within minutes or hours. I was hoping that the new DMAC would solve this issue, but alas it does not. Booted fine the first time, and corrupted the partitions within 10 minutes. Latest firmware on Fastlane Z3.
A3000/25 Mhz ReAmiga, no accelerator. ZuluSCSI works great with onboard SCSI. Let me know if I can provide any additional testing details.

Douglas (10MARC)

**Comment by mbtaylor1982** [comment]
When creating bug reports can you please be a little clearer with the information, I've now created a template with options to select. 

@10MARC please can you check the below summary is correct.

- Firmware: v0.9
- Machine: ReAmiga 3000
- CPU: 030 @ 25Mhz

**Description**

When the Z3 Fastlane is installed, the partitions on the disks connected to the motherboard SCSI controller will become corrupted within 10 mins of use. The same also happens with the SDMAC rev 2.

**Comment by 10MARC** [comment]
Ok. No idea which firmware is on there honestly. Whatever Stephan had at
AmiWest is what's on there. Maybe .8?

Douglas

**Comment by mbtaylor1982** [comment]
@10MARC  if you download and run the [SDMAC test tool](https://github.com/cdhooper/amiga_sdmac_test/releases/tag/release_0.9) you can check the firmware version you are running providing it is v0.6 or later.

**Comment by 10MARC** [comment]
Hey that's great! I will check it out right away. I need to solder a JTAG
header on mine I suppose to update it.

**Comment by 10MARC** [comment]
Just so we have all the information: Here is the output of the SDMAC test.
So it looks like I have .7 and need .9 - time to solder on those JTAG
headers. I think I have the hardware to flash it, but I will have to check.


Memory controller:   Ramsey-04 $d
Ramsey config:       1Mx4, 238 clock refresh
SCSI DMA Controller: SDMAC-04  v0.7
SCSI Controller:     WD33C93A 00-06 or AM33C93A microcode 08, 14.3 MHz
WDC Configuration:   Polled Mode, 245 msec timeout, Offset 12, Sync 4.772
MHz

Ramsey test:  PASS
SDMAC test:   PASS
WDC test:     PASS

Douglas Compton
Dynamic Computing Technologies
520-222-9795
***@***.***
www.dctechaz.com

**Comment by terriblefire** [comment]
Indeed. For Z3 SCSI the DMA is done by the DMA controller on the Z3 Fastlane. Not the (S)DMAC.

**Comment by 10MARC** [comment]
That makes sense to me. I seem to recall people having better luck with the Z3 SCSI with newer chip versions, but that could have just been a coincidence. Thanks for the clarification! Now time to solder on a JTAG header and update my little guy!

**Comment by mbtaylor1982** [comment]
@10MARC are we ok to close this issue down?

**Comment by 10MARC** [comment]
Oh sure! Thanks for all your input. I will report other finding to you as I discover them. The Discord invite does seem broken, though
